### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by Morganov**

### CODE REVIEW

File: book/A-git-in-other-environments/sections/powershell.asc [hunk 1]
```diff
@@ -49,26 +56,31 @@ Then you can go back and try again.
 This happens, because the modules that ship with Windows PowerShell are signed with a different publishment certificate.
 
 ===== Update PowerShell Prompt
+
 To include git information in your prompt, the posh-git module needs to be imported.
-To have posh-git imported every time PowerShell starts, execute the Add-PoshGitToProfile command which will add the import statement into you $profile script.
+To have posh-git imported every time PowerShell starts, execute the `Add-PoshGitToProfile` command which will add the import statement into you `$profile` script.
```
Comment (ben) [inline book/A-git-in-other-environments/sections/powershell.asc:61]: Here's another typo I noticed while reviewing this. Mind including this in your fix?

```suggestion
To have posh-git imported every time PowerShell starts, execute the `Add-PoshGitToProfile` command which will add the import statement into your `$profile` script.
```

File: book/A-git-in-other-environments/sections/zsh.asc [hunk 2]
```diff
@@ -37,14 +37,12 @@ This results in a display of the current branch on the right-hand side of the te
...
 It looks a bit like this:
 
-.Customized `zsh` prompt.
+.Customized `zsh` prompt
 image::images/zsh-prompt.png[Customized `zsh` prompt.]
 
-For more information on vcs_info, check out its documentation
-        in the `zshcontrib(1)` manual page,
-        or online at http://zsh.sourceforge.net/Doc/Release/User-Contributions.html#Version-Control-Information[].
+For more information on `vcs_info`, check out its documentation in the `zshcontrib(1)` manual page, or online at http://zsh.sourceforge.net/Doc/Release/User-Contributions.html#Version-Control-Information[].
```
Comment (ben) [inline book/A-git-in-other-environments/sections/zsh.asc:43]: 😍

File: book/A-git-in-other-environments/sections/powershell.asc (hunk 1 above)
Comment (Morganov) [reply book/A-git-in-other-environments/sections/powershell.asc]: 🤦

**Comment by ben** [comment]
:sparkles: Thanks!

**Pull request merged**